.github/release-drafter.yml

```diff
@@ -0,0 +1,72 @@
+name-template: 'v$RESOLVED_VERSION'
+tag-template: 'v$RESOLVED_VERSION'
+categories:
+  - title: 'Breaking 🔨'
+    label: 'type: breaking'
+  - title: 'New 🚀'
+    label: 'type: feature'
+  - title: 'Bug Fixes 🐛'
+    label: 'type: bug'
+  - title: 'Maintenance ⚙️'
+    label: 'type: maintenance'
+  - title: 'Documentation 📄'
+    label: 'type: documentation'
+  - title: 'Other changes'
+  - title: 'Dependency Updates 💪'
+    label: 'type: dependencies'
+    collapse-after: 5
+
+version-resolver:
+  major:
+    labels:
+      - 'type: breaking'
+  minor:
+    labels:
+      - 'type: feature'
+  patch:
+    labels:
+      - 'type: bug'
```

Comment:
Do these categories suffice or are they overly-granular? Open to discussion here 🙏🏻

Reply:
I think they look fine. I was thinking it would be nice to have a `refactor:` PR prefix and matching `cleanup` label, for code refactoring changes without any real functionality change. Wdyt?

Reply:
Ah yeah, I have that categorised under `chore: ` PR prefix and matches `type: maintenance` label. Would that suffice?